examples/simple_counter/shared/src/app.rs

```diff
@@ -24,16 +26,16 @@ pub struct Model {
     count: isize,
 }
 
-#[derive(Serialize, Deserialize, Clone, Default)]
+#[derive(Facet, Serialize, Deserialize, Clone, Default)]
 pub struct ViewModel {
     pub count: String,
 }
 
 #[derive(Default)]
-pub struct Counter;
+pub struct App;
```

Comment:
Moving away from naming the app types meaningfully? I've probably got a minor preference for `Counter` or `Weather` over just `App`, but no strong opinions.

Reply:
I don't really have a preference so will rename it ;-)